client/src/components/EventView/index.js

```diff
@@ -1,7 +1,174 @@
+    color: black;
+    font-weight: bold;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+`;
+
+const CenterHeadingLight = styled.p`
+    font-size: calc(7px + 2vmin);
+    color: black;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+`;
+
+const Heading = styled.p`
+    font-size: calc(10px + 2vmin);
+    color: black;
+    font-weight: bold;
+`;
+
+const DisplayStle = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: left;
+    align-items: left;
+    width: 70%;
+    height: 100%;
+    padding: 10% 15%;
+`;
+
+const InviteeRow = styled.div`
+  display: flex;
+  flex-direction: row;
+  //align-items: center;
+  align-items:flex-start;
+  //justify-content: space-around;
+`;
+
+const Main = () => {
+  const [apiStatus, setApiStatus] = useState({
+    loading: false,
+    error: false,
+    message: false
+  });
+
+    const [event, setEvent] = useState({
+        eventName: "testName"
+
+    });
+    useEffect(() => {
+      loadPageData();
+      // eslint-disable-next-line
+    }, []);
+
+    //not sure how to implement this
+    // useEffect(() => {
+    //     const params = new URLSearchParams(event.location.search);
+    //     let eventId = params.get("id");
+    //     if (eventId) {
+    //     loadPageData(eventId);
+    //     } else {
+    //     //redirect to home page since there isn't an eventId
+    //     }
+    //     // eslint-disable-next-line
+    //     }, [location.search]);
+    //please help Morgan!!
+
+  const loadPageData = async () => {
+    setApiStatus({ ...apiStatus, loading: true, error: false });
+    const newApiStatus = { ...apiStatus };
+    try {
+        let eventId = "12346"
+      //const results = await ClientService.events();
+      const results = await ClientService.event(eventId);
+      //TODO: call endpoint for invites
+      if (results.success) {
+        //prepareData(results.events);
+        
+        setEvent(results.event);
+      } else {
+        // default message just in case
+        
+        newApiStatus.error = true;
+        newApiStatus.message = results.message;
+      }
+    } catch (error) {
+      newApiStatus.error = true;
+      newApiStatus.message = error.message;
+    }
+    newApiStatus.loading = false;
+
+    setApiStatus(newApiStatus);
+  };
+
+//const Main = () => {
+  return (
+    <div>
+      <AppBar color={eventPink} name = {event.eventName} date = {event.date}>
+      </AppBar>
+      <div>
+      {apiStatus.loading ? (
+        <CenteredLoadingSpinner
+          size={150}
+          color={theme1}
+          loading={apiStatus.loading}
+        />
+      ) : (
+        <Stack gapSize={spacing64}>
+          <DisplayStle>
+        <Heading>Description</Heading>
+        <p>{event.eventName}</p>
+        <RSVPFloating/>
+        <Heading>Location</Heading>
+        <Map>{event.location}</Map>
```

Comment:
For the map, you need to refer to the Map/index.js to see what is being passed to it, and you'll need to properly pass those as props to <Map> and not as children. (the way you have {event.location} right now would be passing down a child element). 

Also, you will need the MapService to convert the address from the event.location.address to the expected form that maps need.  (I can help you more with this, but try to get it started)